package.json

```diff
@@ -18,15 +18,19 @@
   "scripts": {
     "install": "node-pre-gyp install --fallback-to-build",
     "clean": "node-pre-gyp clean",
-    "test": "tap -Rspec test/*.js"
+    "test": "tape test/*.test.js"
```

Comment:
Why the change?

Reply:
The node-cpp-skel (which this project was first based on) framework assumes `tape` instead of `tap`.  They're much the same, but using `tape` will make life simpler in the future if we pull in updates from `node-cpp-skel`.